warp10/src/main/java/io/warp10/worf/EncodeTokenCommand.java

```diff
@@ -59,7 +66,19 @@ public boolean execute(String function, WorfKeyMaster worfKeyMaster, PrintWriter
         out.println("token=" + token);
         out.println("tokenIdent=" + worfKeyMaster.getTokenIdent(token));
         out.println("application name=" + application);
-        out.println("producer & owner=" + producer + " & " + owner);
+        out.println("producer=" + producer);
+
+        switch (tokenType) {
+          case READ:
+            out.println("authorizations: applications" + applications);
```

Comment:
No space between the text and the list?

Reply:
Updated with = separator (output homogeneity)